extension/src/test/fixtures/expShow/survival/rows.ts

```diff
@@ -159,6 +160,7 @@ const data: Commit[] = [
     starred: false
   },
   {
+    branch: 'current',
```

Comment:
I had "main" at first but changed it to "current" for the webview tests. It was mainly because of these tests (but there are other ones:

```
 describe('Sub-rows middle states indicators', () => {
    const testRowLabel = 'main'

    const getMiddleStatesTestRow = () => {
      return getRow(testRowLabel)
    }

   ...
  })
```

The name of the branch should not really matter either. But if you feel strongly about this, I don't mind changing the branch name and finding something to target the main rev row.